openprescribing/frontend/tests/commands/test_import_measures_new.py

```diff
@@ -585,3 +320,271 @@ def validate_practice_statistics_measure_value(self, mv, series):
         else:
             self.assertAlmostEqual(mv.calc_value, series['ratio'])
             self.assertAlmostEqual(mv.percentile, series['ratio_percentile'])
+
+
+def create_organisations():
+    '''Create RegionalTeams, STPs, CCGs, Practices'''
+
+    for regtm_ix in range(5):
+        regtm = RegionalTeam.objects.create(
+            code='Y0{}'.format(regtm_ix), name='Region {}'.format(regtm_ix)
+        )
+
+        for stp_ix in range(5):
+            stp = STP.objects.create(
+                ons_code='E000000{}{}'.format(regtm_ix, stp_ix),
+                name='STP {}'.format(regtm_ix, stp_ix),
+            )
+
+            for ccg_ix in range(5):
+                ccg = PCT.objects.create(
+                    regional_team=regtm,
+                    stp=stp,
+                    code='{}{}{}'.format(regtm_ix, stp_ix, ccg_ix).replace(
+                        '0', 'A'
+                    ),
+                    name='CCG {}/{}/{}'.format(regtm_ix, stp_ix, ccg_ix),
+                    org_type='CCG',
+                )
+
+                for prac_ix in range(5):
+                    Practice.objects.create(
+                        ccg=ccg,
+                        code='P0{}{}{}{}'.format(regtm_ix, stp_ix, ccg_ix, prac_ix),
+                        name='Practice {}/{}/{}/{}'.format(
+                            regtm_ix, stp_ix, ccg_ix, prac_ix
+                        ),
+                        setting=4,
+                    )
+
+
+def set_up_bq():
+    '''Set up BQ, and upload CCGs and practices.'''
+
+    Client('measures').create_dataset()
+    client = Client('hscic')
+    table = client.get_or_create_table('ccgs', schemas.CCG_SCHEMA)
+    columns = [field.name for field in schemas.CCG_SCHEMA]
+    table.insert_rows_from_pg(PCT, columns, schemas.ccgs_transform)
```

Comment:
While we're bothering to move things around for readability, it would be nice to split this into `set_up_bq` and `upload_ccgs_and_practices`.